templates/circuit-breaker.j2

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+##
+## Requirements:
+## -------------
+##
+## - fping
+## - iptables
+
+{% set minio_nodes = minio_server_cluster_nodes | join('" "') | replace("http://", "") | regex_replace ('\/[a-zA-Z0-9]+', "") | regex_replace (':[0-9]+', "") %}
```

Comment:
This line is pure magic, I think that an example of what it does would be useful for future user.

Reply:
Done